### PR TITLE
fix: lys hub show only incompleted and recent tasks

### PR DESCRIPTION
**Pull request by rjchow**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

- Modifies the LYS Tasklist display such that only tasks that are either incomplete or recently actioned from within LYS are shown

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Enable the `launch-your-store` feature flag
2. Prior to entering LYS, complete at least one task from within Add Products, Customize your store, Payments, Shipping, or Taxes
3. Enter LYS
4. Observe that the task you completed previously will not show up on the LYS task list
5. Click on one of the LYS task list tasks and complete it
6. Return to LYS
7. Observe that the task shows as strikethrough and becomes non-interactive
8. This feature has a 7 day validity period so the recently actioned tasks will not show up if it has been more than 7 days.

### Changelog entry

-   [ ] Automatically create a changelog entry from the details below.

<details>

#### Significance

-   [ ] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Comment 

</details>

### CODE REVIEW

File: plugins/woocommerce-admin/client/launch-your-store/hub/sidebar/tasklist.tsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+	) as string[];
+
+	const tasklist = await resolveSelect(
+		ONBOARDING_STORE_NAME
+	).getTaskListsByIds( [ 'setup' ] );
+
+	const recentlyActionedTasks = getRecentlyActionedTasks() ?? [];
+
+	/**
+	 * show tasks that fulfill all the following conditions:
```
Comment (ilyasfoo) [inline plugins/woocommerce-admin/client/launch-your-store/hub/sidebar/tasklist.tsx]: Super nit

```suggestion
	 * Show tasks that fulfill all the following conditions:
```

**Review by ilyasfoo — APPROVED** [review]
According to the Figma (0BWMerqRR1f468EwC7g5tg-fi-835_28135), the font color for completed tasks looks more pale:

image

Otherwise this is testing well, pre-approving 🚢

**Comment by rjchow** [comment]
> According to the Figma ([0BWMerqRR1f468EwC7g5tg-fi-835_28135](https://href.li/?https://www.figma.com/file/0BWMerqRR1f468EwC7g5tg/?node-id=835-28135)), the font color for completed tasks looks more pale:
> image
> 
> Otherwise this is testing well, pre-approving 🚢

I dunno how you managed to see that 🤣 Thanks, fixed!

**Review by psealock — APPROVED** [review]
Code is looking good and this is testing well 🚀

**Pull request merged**